CHANGELOG.rst

```diff
@@ -11,6 +11,13 @@ Added
 - Support "instructions" to perform a match.
 - Support instruction type `goto_table`.
 
+[2024.2.0] - 2024-02-06
+***********************
+
+Added
+=====
+- Added UI to sdntrace_cp.
```

Comment:
Great to have it documented on changelog, but could you move it this under the `UNRELEASED] - Under development` section? During development we typically add then, and those will become `2023.2.0` when we bump the `2023.2` version soon.